pydy/codegen/tests/test_code.py

```diff
@@ -98,7 +98,7 @@ def test_generate_ode_function(self):
         expected_dx = np.array([v, 1.0 / m * (-c * v + m * g - k * x + F)])
 
         rhs_args = {'constants': constants_map,
-                    'specified': {specifieds[0]: F}}
+                    'specified': {next(iter(specifieds)): F}}
```

Comment:
I don't like the use of set because it is unordered. I believe all of the code printing relies on the constants, coordinates, speeds, and specifieds to be ordered. It is confusing enough to keep track of them as it is, and guaranteeing order on these public attributes is important.